### DISCUSSION TITLE
Button actions

### DISCUSSION THREAD

**Discussion by alidaf** [sle118/squeezelite-esp32#385] [Q&A]
Can somebody please shed some light on what the BCTRLS_PSn actions relate to.

**Comment by Renber78** [comment]
These are the preselection keys

**Reply by alidaf** [reply]
Preselection?

**Reply by Renber78** [reply]
Sorry, it's a translation...
Preset key
It's the same as the 6 keys next to the screen on the squeezebox radio for example.

**Reply by alidaf** [reply]
Ah, that makes sense. Thank you.
